src/Summoner/ProjectData.hs

```diff
@@ -1,12 +1,22 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric  #-}
+{-# LANGUAGE InstanceSigs   #-}
```

Comment:
I think `DeriveGeneric` and `InstanceSigs` should be put into default extensions safely.